debugger/io.c

```diff
@@ -19,6 +19,20 @@
 #include "io.h"
 #include "websockets/packet_queue.h"
 
+#define BIT_COUNT 8
+
+static void digitalIoToString(char* const output, const uint8_t mask, const uint8_t x)
```

Comment:
one of rare occasions to use  known output size: `char output[BIT_COUNT + 1] `, should allow compiler to detect some bugs (hopefully)

Reply:
Ack